api/api.js

```diff
@@ -8,14 +7,6 @@ import routes from './routes';
 
 const app = express();
 
```

Comment:
```
Warning: connect.session() MemoryStore is not
designed for a production environment, as it will leak
memory, and will not scale past a single process.

``` 
We are seeing this message in production, which might be another reason we have issues with the app. Might prevent memory leak. 

We don't don't use sessions and until we implement a redis session store - we shouldn't use a memory store.